pos_order_mgmt/models/pos_config.py

```diff
@@ -10,26 +11,31 @@
 class PosConfig(models.Model):
     _inherit = 'pos.config'
 
+    iface_order_mgmt = fields.Boolean(
+        string='Order Management',
```

Comment:
Should be default to True.
- updating the module will disable the feature otherwise.
- when installing the module, it is expected the feature is enabled.

Reply:
Yeah, you're right

Done